assets/css/calypsoify.css

```diff
@@ -316,6 +316,105 @@ table.widefat {
 	color: #2E4453;
 }
 
+/* Forms */
+
+#post-body-content {
+	background-color: #fff;
+	box-shadow: 0 0 0 1px rgba(200,215,225,0.5), 0 1px 2px #e9eff3;
+	box-sizing: border-box;
+	padding: 24px;
+}
+
+#title-prompt-text {
+	clip: initial !important; /*avoid clipping when field is on focus */
+	clip-path: initial;
+	color: #2e4453 !important;
+	display: block;
+	font-size: 14px !important;
+	font-weight: 600;
+	height: auto;
+	margin: 0 0 5px 0 !important;
+	padding: 0 !important;
+	position: static !important;
+	width: auto;
+}
+
+#wp-content-editor-tools {
+	background: #fff;
+}
+
+/* Adds card layout style to forms */
+
+#mainform h2,
+#mainform h3 {
+	background-color: #fff;
+	border: 1px solid rgba(200,215,225,0.5);
```

Comment:
Do these need to be rgba or can they be hex?

Reply:
I copied the value from .nav-tab-wrapper. But we can use #e4eaef.